src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
 // write code here
+const tree = document.querySelector('.tree');
+const headlines = document.querySelectorAll('li');
+
+function wrapTextToSpan(element) {
+  const children = [...element.children];
+
+  for (const child of element.children) {
+    child.remove();
+  }
+
+  const copyText = element.innerText.trim();
+  const newSpan = document.createElement('span');
+
+  element.textContent = '';
+  newSpan.innerText = copyText;
+  element.appendChild(newSpan);
+
+  children.forEach((item) => element.appendChild(item));
+}
+
+for (const el of headlines) {
+  if ([...el.children].length > 0) {
+    wrapTextToSpan(el);
+  }
+}
+
+tree.addEventListener(
+  'click',
+  (ev) => {
+    if (ev.target.tagName === 'SPAN') {
+      for (const child of ev.target.closest('li').children) {
```

Comment:
You don't need loop here - you always want to apply it to one element

You can get it using [nextSibling](https://developer.mozilla.org/en-US/docs/Web/API/Node/nextSibling) property